SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -459,11 +459,9 @@ void USpatialConnectionManager::SetupConnectionConfigFromURL(const FURL& URL, co
 
 		ReceptionistConfig.WorkerType = SpatialWorkerType;
 
-		const TCHAR* UseExternalIpForBridge = TEXT("useExternalIpForBridge");
-		if (URL.HasOption(UseExternalIpForBridge))
+		if (URL.HasOption(*SpatialConstants::URL_USE_EXTERNAL_IP_FOR_BRIDGE_OPTION))
 		{
-			FString UseExternalIpOption = URL.GetOption(UseExternalIpForBridge, TEXT(""));
-			ReceptionistConfig.UseExternalIp = !UseExternalIpOption.Equals(TEXT("false"), ESearchCase::IgnoreCase);
+			ReceptionistConfig.UseExternalIp = true;
```

Comment:
I changed these for consistency since we use the presence of a URL option as a bool flag everywhere else without requiring a value. Also, options in Unreal URLs can be removed by prefixing them with `-`.
Only boolean command line args actually require a value.